code/datums/commodity.dm

```diff
@@ -2882,3 +2882,821 @@
+	price = 4000
+	upperfluc = 3000
+	lowerfluc = -3000
+/datum/commodity/blackmarket/wrestler/nassa
+	//would like to have the price higher but alas it's avaliable in a QM crate and valued at about 7.5k there
+	comname = "NASSA space suit"
+	comtype = /obj/item/clothing/under/gimmick/blackstronaut
+	desc_sell = "I'd LOVE to get my hands on a couple sets of that prototype jumpsuit. WRESTLING IN THE COLD VACCUM OF SPACE! I'll pay a BUNCH for any you find in good condition!"
+	price = 6200
+	upperfluc = 800
+	lowerfluc = -800
+/datum/commodity/blackmarket/wrestler/uplink_telecrystal
+	//this is the bad guy tokens, and it's a lowball offer. this is primarily for using telescrystals to bribe crew members (who have no legitimate use for TCs)
+	comname = "Pure Telecrystal Token"
+	comtype = /obj/item/uplink_telecrystal
+	desc_sell = "These things are practically CURRENCY on some planets! I can get you a GREAT DEAL on them!"
+	price = 30000
+	upperfluc = 1
+	lowerfluc = -1
+/datum/commodity/blackmarket/wrestler/sacred_sandal
+	comname = "Sacred Sandals"
+	comtype = /obj/item/clothing/shoes/hermes
+	desc_sell = "These things have GOT TO BE ORIGINALS! Not the knock off "Magic Sandals" the space wizards manufacture in sweatshops!"
+	price = 500000
+	upperfluc = 100000
+	lowerfluc = -100000
+/datum/commodity/blackmarket/wrestler/magic_sandal
+	//these things are everywhere
+	comname = "Magic Sandals"
+	comtype = /obj/item/clothing/shoes/sandal
+	desc_sell = "That said, those knock offs are DAMN WELL MADE! I'll buy those for a fair price!"
+	price = 6800
+	upperfluc = 1200
+	lowerfluc = -1200
+/datum/commodity/blackmarket/wrestler/glitch_gun
+	//not sure if this is legitimately obtainable?
+	comname = "Glitch Gun"
+	comtype = /obj/item/gun/energy/glitch_gun
+	desc_sell = "You hear about the fabled Glitch Gun? It's said to be able to shoot bullets though code itself. If you find it, SELL IT TO ME!"
+	price = 1620000
+	upperfluc = 1200
+	lowerfluc = -1200
+/datum/commodity/blackmarket/wrestler/capsuit
+	comname = "Captain's Space Suit"
+	comtype = /obj/item/clothing/suit/space/captain
+	desc_sell = "The old style green space suits NT used to issue to their captains are hella valuable, but unfortunately also hella rare. Hey, you're a Traser, aren't ya? Could you get your captain's space suit to me?"
+	price = 605000
+	upperfluc = 50000
+	lowerfluc = -50000
+/datum/commodity/blackmarket/wrestler/dagger
+	//this is both the traitor's dagger and the biodome dagger
+	comname = "Sacrificial Dagger"
+	comtype = /obj/item/dagger
+	desc_sell = "Old knives are great investments! The more occult the better!"
+	price = 210400
+	upperfluc = 30000
+	lowerfluc = -30000
+/datum/commodity/blackmarket/wrestler/shoecrafts
+	//space sweatshop 13
+	comname = "Custom Shoes"
+	comtype = /obj/item/clothing/shoes/crafted
+	desc_sell = "And any shoes YOU HAND CRAFT, I WILL BUY!"
+	price = 800
+	upperfluc = 600
+	lowerfluc = -600
+/datum/commodity/blackmarket/wrestler/katana_nukeop
```

Comment:
(not a problem with code just want to leave a note)
Nuke Op's dont last long enough to reach the time when this trader would become active according to your post on the forums. I'd like to clarify too, that with all my stuff, I'm not a dev. I'm some dumbass who doesnt know how to get out of review mode in git and ended up in it on accident.